### PR TITLE
[FIX] Temporarily changing stderr on Windows because of VS14 bug (develop)

### PR DESCRIPTION
**Pull request by cpockrandt**
See #1613

### CODE REVIEW

**Comment by xenigmax** [comment]
This PR is based on master but the target is to develop. I will check this later.

**Comment by xenigmax** [comment]
@rrahn Can you merge this after checking?

File: apps/pair_align/lib/CMakeLists.txt [hunk 1]
```diff
@@ -32,7 +32,7 @@ foreach(GAPS_T ${FLAG_VALUES})
 				# Build library from source -> Note: STATIC could be replaced with OBJECT only building 
 				# the object files not linking them into a lirary. However this feature
 				# is shipped with cmake 2.8.8, which is not supported at the moment.
-				add_library(${TARGET} STATIC pair_align_global.cpp)
+                add_library(${TARGET} OBJECT pair_align_global.cpp)
```
Comment (rrahn) [inline apps/pair_align/lib/CMakeLists.txt]: The indentation seems to be off.

File: apps/pair_align/lib/CMakeLists.txt [hunk 2]
```diff
@@ -41,7 +41,7 @@ foreach(GAPS_T ${FLAG_VALUES})
 				set_property (TARGET ${TARGET} APPEND PROPERTY COMPILE_DEFINITIONS "SUFFIX_GAP_BOTTOM=${GAPS_B}")
 				
 				# Add target name to lib targets.
-				list(APPEND LIB_TARGETS ${TARGET})
+                list(APPEND LIB_TARGETS "$<TARGET_OBJECTS:${TARGET}>")
```
Comment (rrahn) [inline apps/pair_align/lib/CMakeLists.txt]: Here as well? Rather stick to the old whitespaces, if it is only one line.

**Comment by rrahn** [comment]
Never mind the issues above. I assume they have been already merged :disappointed:

**Pull request merged**